plugin/evm/limitorders/memory_database.go

```diff
@@ -579,17 +579,15 @@ func (db *InMemoryDatabase) getReduceOnlyOrderDisplay(order *LimitOrder) *LimitO
 		if position.Size.Sign() == order.BaseAssetQuantity.Sign() {
```

Comment:
should this be
```
if position.Size.Sign == 0 || (position.Size.Sign() == order.BaseAssetQuantity.Sign()) {
    return nil
}
```

Reply:
right, cuz we don't delete position if it becomes 0